backend/sozisel/lib/sozisel_web/schema/mutations/presenter_mutations.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SoziselWeb.Schema.Mutations.PresenterMutations do
+  use SoziselWeb.Schema.Notation
+
+  alias SoziselWeb.Schema.Resolvers.PresenterResolvers
+  alias SoziselWeb.Schema.{Middleware}
+
+  object :presenter_mutations do
+    field :send_event_to_all, :launched_event do
```

Comment:
As we have a schema that is called launched event this probably should be named `launch_event`.

You can have a parameter `broadcast` that is a boolean indicating that it will bo send to everyone and `target_participants` that will be a array of string indicating participant ids to which send the event. If broadcast is set to true then just ignore the participants array, if set to false then just use the target participants array